bittide/src/Bittide/Calendar.hs

```diff
@@ -38,40 +43,41 @@ it does not care about the type of its entries, this type depends on the compone
 instantiates the calendar.
 -}
 
+type ValidEntry a repetitionBits = (a, Unsigned repetitionBits)
+type Calendar size a repetitionBits  = Vec size (ValidEntry a repetitionBits)
```

Comment:
For a next PR: use memblob. This is probably going to wreak havoc at some point.

Reply:
These types need documentation. Most importantly, they'll need to point out that entries can be repeated.